resources/tentacle.rb

```diff
@@ -37,3 +37,5 @@
 attribute :api_key, kind_of: String
 attribute :roles, kind_of: Array
 attribute :environment, kind_of: String, default: node.chef_environment
+attribute :tenant, kind_of: Array
```

Comment:
Empty array gives this deprecation warning: 

defaulting to nil is fine. 

```
       Deprecated features used!
         An attempt was made to change tenant_tags from [] to nil by calling tenant_tags(nil). In Chef 12, this does a get rather than a set. In Chef 13, this will change to set the value to nil. at 1 location:
           - C:/Users/ADMINI~1/AppData/Local/Temp/kitchen/cache/cookbooks/nd-octopus/recipes/tentacle.rb:15:in `block in from_file'
```